tests/apikey_test.go

```diff
@@ -0,0 +1,14 @@
+package easypost_test
+
+func (c *ClientTests) TestUserApiKeysAll() {
+	client := c.ProdClient()
+	assert := c.Assert()
+
+	apiKeys, err := client.GetAPIKeys()
+	if err != nil {
+		c.T().Error(err)
+		return
+	}
```

Comment:
Please look at the new style for checking no error occurred (my last PR for this lib) and use that here instead.